### PR TITLE
Do not start services if they have timers

### LINKED ISSUE #13: Do not start a service if it has a timer
To make sure we don't accidentally trigger something untimely. We just need to make sure we started the `.timer` portion of the `.service` and return `true`

### PR DESCRIPTION
**Pull request by DeviaVir**
If we always start a service, even if it has a timer, we may start a service at a moment that it's unwanted.

Fixes #13

### CODE REVIEW

File: src/test/agent/test_systemd.py [hunk 1]
```diff
@@ -1,69 +1,85 @@
...
-        self.manager.load_unit.return_value = self.unit
+        self.manager.load_unit.side_effect = [self.unit1, self.unit2,
+                                              self.unit3]
 
         units = [u for u in self.systemd._get_units(self.manifest)]
-        self.assertEquals(1, len(units))
-        self.assertEquals(self.unit, units[0])
+        self.assertEquals(2, len(units))
+        self.assertEquals(self.unit1, units[0])
+        self.assertEquals(self.unit2, units[1])
```
Comment (DeviaVir) [inline src/test/agent/test_systemd.py]: @thepwagner was hoping to catch some feedback here:
I don't understand why it's returning `self.unit2` (the 'skipped' unit) but does not return the `bar.timer` unit..

Comment (thepwagner) [inline src/test/agent/test_systemd.py]: It's returning from https://github.com/pebble/spacel-agent/pull/14/files#diff-da8ae053713a8bea00a66233916e12d2R61 ; but your added check only covers https://github.com/pebble/spacel-agent/pull/14/files#diff-da8ae053713a8bea00a66233916e12d2R67

There's a unique path for if systemd already has `bar.service` loaded, or if it needs to JIT load it. Mocking `list_units()` is the former.

File: src/test/agent/test_systemd.py [hunk 2]
```diff
@@ -1,69 +1,85 @@
...
+TIMER_NAME = 'bar.timer'
 
 
 class TestSystemdUnits(unittest.TestCase):
     def setUp(self):
-        self.unit = MagicMock()
-        self.unit.properties.Id = SYSTEMD_SERVICE
+        self.unit1 = MagicMock()
+        self.unit1.properties.Id = SYSTEMD_SERVICE
+        self.unit2 = MagicMock()
```
Comment (thepwagner) [inline src/test/agent/test_systemd.py]: Naming: `timer_service` would make later uses more readable than remembering which unit{1,3} is which

File: src/test/agent/test_systemd.py (hunk 1 above)
Comment (DeviaVir) [inline src/test/agent/test_systemd.py]: ah, yes, of course! thanks 💃

**Comment by DeviaVir** [comment]
screen shot 2016-07-19 at 10 54 42

**Comment by DeviaVir** [comment]
@thepwagner @lrvick going to merge this in, tested this branch successfully and want to make sure I don't accidentally trigger something on prod that should not be triggered right now without changing all of them to this branch, open for retrospective feedback, of course.

**Pull request merged**